mlir/lib/Conversion/MemRefToSPIRV/MemRefToSPIRV.cpp

```diff
@@ -623,7 +631,8 @@ LoadOpPattern::matchAndRewrite(memref::LoadOp loadOp, OpAdaptor adaptor,
   if (!loadPtr)
     return failure();
 
-  AlignmentRequirements requiredAlignment = calculateRequiredAlignment(loadPtr);
+  AlignmentRequirements requiredAlignment =
```

Comment:
This function is calculating the alignement; non termporal is not about the alignment. So I'm not sure we should put it here?

Reply:
```c++
using AlignmentRequirements = FailureOr<std::pair<spirv::MemoryAccessAttr, IntegerAttr>>;
```

It does calculate both memory access and alignment and nontemporal is `spirv::MemoryAccess::Nontemporal`, so I think it fits.

Reply:
Yeah renaming it would be good. As the function name shows it's about alignment calculation right now.

Reply:
Done 👍